lms/envs/common.py

```diff
@@ -2561,3 +2561,7 @@
 # Number of seconds before JWT tokens expire
 JWT_EXPIRATION = 30
 JWT_ISSUER = None
+
+# Credit notifications settings
+NOTIFICATION_EMAIL_CSS = "templates/credit_notifications/credit_notification.css"
+NOTIFICATION_EMAIL_EDX_LOGO = "templates/credit_notifications/edx-logo-header.png"
```

Comment:
We need to be careful about using EdX brand images by default, since this will get used in Open Source installations as well.  Elsewhere, we default to an OpenEdX logo and override it in our production environments to point to an EdX brand logo.

For example, the logo used in the footer is set here: https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L1077

But can be overridden because: https://github.com/edx/edx-platform/blob/master/lms/envs/aws.py#L329